pkgs/applications/audio/audacity/default.nix

```diff
@@ -66,31 +66,28 @@
 let
   inherit (lib) optionals;
   pname = "audacity";
-  version = "3.1.3";
+  version = "3.2.1";
 
-  wxWidgets_src = fetchFromGitHub {
-    owner = pname;
-    repo = "wxWidgets";
-    rev = "v${version}-${pname}";
-    sha256 = "sha256-KrmYYv23DHBYKIuxMYBioCQ2e4KWdgmuREnimtm0XNU=";
-    fetchSubmodules = true;
-  };
-
-  wxGTK' = wxGTK.overrideAttrs (oldAttrs: rec {
-    src = wxWidgets_src;
+  wxGTK' = if stdenv.isDarwin then wxGTK31 else
+  wxGTK31.overrideAttrs (oldAttrs: rec {
+    version = "3.1.3.1-${pname}";
```

Comment:
```suggestion
    version = "3.1.3.1";
```

Reply:
It seemed to me that most distributions target the official wxGTK31 [(Arch)](https://archlinux.org/packages/community/x86_64/audacity/) / wxGTK32 [(Debian)](https://packages.debian.org/fr/sid/audacity) instead of the Audacity's fork, so I pushed another commit to target wxGTK32.